### PR TITLE
Expands sandbox setup documentation

### PR DESCRIPTION
**Pull request by matthugs**
Won't require a release! Or anything! What great fun.

### CODE REVIEW

File: docs/developing.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ edX Proctoring Developer Guide
 
 
 How do I use proctoring on devstack?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+------------------------------------
```
Comment (matthugs) [inline docs/developing.rst]: By way of explanation, we skipped a section heading level I found I wanted to use

File: docs/developing.rst [hunk 2]
```diff
@@ -119,8 +123,79 @@ Start by following the steps here: https://github.com/edx/edx-proctoring
...
+Proctortrack
+""""""""""""
+
+As will be the case with all REST backends implementing `our spec`_, one
+doesn't need to configure much to get Proctortrack working on a
+sandbox, e.g.::
+  EDXAPP_PROCTORING_BACKENDS:
+    DEFAULT: 'proctortrack'
+    proctortrack:
+      client_id: "<you'll need to fill this these in wth credentials from Proctortrack>"
```
Comment (MichaelRoytman) [inline docs/developing.rst]:
```suggestion
      client_id: "<you'll need to fill this in with credentials from Proctortrack>"
```

File: docs/developing.rst [hunk 3]
```diff
@@ -119,8 +123,79 @@ Start by following the steps here: https://github.com/edx/edx-proctoring
...
+      client_secret: "<you'll need to fill this these in wth credentials from Proctortrack>"
```
Comment (MichaelRoytman) [inline docs/developing.rst]:
```suggestion
      client_secret: "<you'll need to fill this in with credentials from Proctortrack>"
```

File: docs/developing.rst [hunk 4]
```diff
@@ -119,8 +123,79 @@ Start by following the steps here: https://github.com/edx/edx-proctoring
...
 
+Backend-specific Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One of the main motivations for setting up a sandbox to test
+proctoring is having an externally accessible system which can be
+accessed by our proctoring providers' systems. This enables more
+thorough end-to-end testing.
+
+To enable proctoring in a way that won't be overridden by ansible
```
Comment (MichaelRoytman) [inline docs/developing.rst]: After one sshing to the running sandbox and updating the yaml file, is any action required after that change?  Do you need to rerun ansible on the sandbox (e.g. as describe in the `Updating code` section of [this](https://openedx.atlassian.net/wiki/spaces/EdxOps/pages/13960183/Sandboxes) document)? Or does that occur as part of restarting the lms and cms processes?

File: docs/developing.rst [hunk 5]
```diff
@@ -119,8 +123,79 @@ Start by following the steps here: https://github.com/edx/edx-proctoring
...
+  EDXAPP_PROCTORING_BACKENDS:
+    ...
+
+Placing these configurations here (rather than the more generic
+locations mentioned in `the README`_) will allow us to leverage the
+power of the ansible playbooks used to construct and administer
+sandboxes.
+
+The contents of ``EDXAPP_PROCTORING_BACKENDS`` will depend on which
+backend you're interested in testing. It's nececssary to provide a
```
Comment (MichaelRoytman) [inline docs/developing.rst]:
```suggestion
backends you're interested in testing. It's necessary to provide a
```
Re: `backend` versus `backends`, the way this is phrased implies, to me, that the sandbox can be configured with only one backend. I think making backends plural clarifies that you can configure your sandbox with more than one backend, which may be a helpful distinction for people new to proctoring.

File: docs/developing.rst (hunk 4 above)
Comment (matthugs) [reply docs/developing.rst]: @MichaelRoytman do you think the update I put for this is enough?

Comment (MichaelRoytman) [reply docs/developing.rst]: I think so! The only thing I notice is that the bullet points under `In addition to adding these configurations, you'll also need to set up
a user which PT can authenticate as.` aren't displaying correctly when viewing the file in GitHub.

Comment (matthugs) [reply docs/developing.rst]: Good eye, thanks!

**Pull request merged**